danger_code_ws/1.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
```

Comment:
Consider adding a namespace to this code to prevent potential naming conflicts.  This is especially important as the codebase grows.